kilink/templates/helpers.html

```diff
@@ -64,7 +106,6 @@
     <script src='/static/mode/python/python.js'></script>
     <script src='/static/mode/r/r.js'></script>
     <script src='/static/mode/ruby/ruby.js'></script>
-<!--     <script src='/static/mode/rust/rust.js'></script> -->
```

Comment:
why 'rust' was commented out?

Reply:
In a previous PR, rust mode stops working after we update the CodeMirror lib.